{"name": "test_error_app_key", "status": "passed", "description": "错误的app_key参数", "start": 1599382205800, "stop": 1599382205832, "uuid": "b8cd6bae-5f77-43ad-ab07-4d1fceca086b", "historyId": "456ba0e2a8e490d60babd01b2b6bb235", "testCaseId": "b200633efbe84b70d71eaefc1cc76b1d", "fullName": "case.test_user_login.Login_user#test_error_app_key", "labels": [{"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "Login_user"}, {"name": "host", "value": "V_LFJJLIU-NB0"}, {"name": "thread", "value": "7516-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_user_login"}]}